data/src/main/java/com/bunbeauty/data/repository/AdditionRepository.kt

```diff
@@ -38,11 +60,35 @@ class AdditionRepository @Inject constructor(
             additionUuid = additionUuid,
             additionPatchServer = updateAddition.mapUpdateAdditionServerToPatchAddition(),
             token = token
-        )
+        ).onSuccess {
+            updateLocalCache(
```

Comment:
Patch returns `AdditionServer`. 
We need to map this response from server and put it to `additionListCache`. 
Do not use `UpdateAddition` to update cache

Reply:
Fixed